cmd/commands/utils_test.go

```diff
@@ -54,27 +45,16 @@ func TestParse(t *testing.T) {
 		return coin
 	}
 
-	//define the list of coin tests
-	var testList = []struct {
-		testPass bool
-		errMsg   string
-	}{
-		//testing ParseCoin Function
-		{types.Coin{} == makeCoin(""), "parseCoin makes bad empty coin"},
-		{types.Coin{"fooCoin", 1} == makeCoin("1fooCoin"), "parseCoin makes bad coins"},
-		{types.Coin{"barCoin", 10} == makeCoin("10 barCoin"), "parseCoin makes bad coins"},
-
-		//testing ParseCoins Function
-		{types.Coins{{"fooCoin", 1}}.IsEqual(makeCoins("1fooCoin")),
-			"parseCoins doesn't parse a single coin"},
-		{types.Coins{{"barCoin", 99}, {"fooCoin", 1}}.IsEqual(makeCoins("99barCoin,1fooCoin")),
-			"parseCoins doesn't properly parse two coins"},
-		{types.Coins{{"barCoin", 99}, {"fooCoin", 1}}.IsEqual(makeCoins("99 barCoin, 1 fooCoin")),
-			"parseCoins doesn't properly parse two coins which use spaces"},
-	}
-
-	//execute the tests
-	for _, tl := range testList {
-		assert.True(t, tl.testPass, tl.errMsg)
-	}
+	//testing ParseCoin Function
```

Comment:
not really, makeCoin is a function variable wrap on ParseCoin which checks an error simultaneously